### ISSUE TITLE
Mouse pointer corruption when connecting to XRDP instance

### ISSUE THREAD

**Issue by andrewheberle** [cedrozor/myrtille#297]
We are experiencing "corruption" of the mouse pointer when using Myrtille to connect to a Ubuntu based XRDP server.

Using the native Windows RDP client works fine, but via Myrtille the mouse pointer image is "corrupted" (garbled could be a better description).

Functionality is still perfect...its just the rendering of the mouse pointer that is the issue.

Screenshots are problematic as the pointer is hidden while taking a screenshot in Windows 10...so this also suggests it is a client site rendering issue somewhere.

Client devices/browsers tested:

- Windows 10 with Edge (new Chromium-based version)
- Windows 10 with Chrome

**Comment by cedrozor** [comment]
Yes, I had noticed it and it also occurs in some cases under Windows RDP (https://github.com/cedrozor/myrtille/issues/161). I already tried to fix it but without success.

I need to resync FreeRDP with the master repository, maybe this was fixed since.

**Comment by dbychkov** [comment]
@cedrozor Is it still not fixed? I see corrupted cursor when editing cells in MS Excel.

**Comment by joemcd0** [comment]
For anyone using Myrtille and having this issue. As the tool isn't in development anymore here is the workaround that works for us. 

A quick experiment shows that using custom cursors can work around this issue.

Firstly copy custom  Cursors  into C:/Windows/Cursors on both the RDS and the Session Host.

Then you can change the cursor on the Session Host for every user in the mouse properties

Alternatively create a group policy to do so.

[Bing Videos](https://www.bing.com/videos/riverview/relatedvideo?pglt=43&q=set+mouse+pointer+group+policy&cvid=6291c270bbbc42488749b082532fc442&gs_lcrp=EgRlZGdlKgYIABBFGDkyBggAEEUYOTIGCAEQABhAMgYIAhAAGEAyBggDEAAYQDIGCAQQABhAMgYIBRAAGEAyBggGEAAYQDIGCAcQABhAMgYICBAAGEAyCAgJEOkHGPxV0gEIODMyMGowajGoAgCwAgA&PC=DCTS&ru=%2fsearch?pglt%3d43%26q%3dset%2bmouse%2bpointer%2bgroup%2bpolicy%26cvid%3d6291c270bbbc42488749b082532fc442%26gs_lcrp%3dEgRlZGdlKgYIABBFGDkyBggAEEUYOTIGCAEQABhAMgYIAhAAGEAyBggDEAAYQDIGCAQQABhAMgYIBRAAGEAyBggGEAAYQDIGCAcQABhAMgYICBAAGEAyCAgJEOkHGPxV0gEIODMyMGowajGoAgCwAgA%26FORM%3dANNAB1%26PC%3dDCTS&mmscn=vwrc&mid=C02637C162ACFA311C6DC02637C162ACFA311C6D&FORM=WRVORC&ntb=1&msockid=14c75a6edee911efb716ee71e7d6ec90)
